core/src/main/resources/schema/app/20220303125000_CB-16079_alter_stackpatch_table_add_status_columns.sql

```diff
@@ -0,0 +1,16 @@
+-- // CB-16079
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stackpatch ADD COLUMN IF NOT EXISTS status CHARACTER VARYING(255);
+UPDATE stackpatch SET status = 'FIXED' WHERE status IS NULL;
+ALTER TABLE stackpatch ALTER COLUMN status SET NOT NULL;
+
+ALTER TABLE stackpatch ADD COLUMN IF NOT EXISTS status_reason TEXT;
+
+-- //@UNDO
+-- SQL to undo the change goes here.
+
+DELETE FROM stackpatch WHERE status != 'FIXED';
```

Comment:
Whereas the `DROP COLUMN IF EXISTS status` can tolerate the absence of the `status`, this `DELETE` will likely fail in that case. For instance, see the output got when running this in cbdb with the dummy column `foobar`:
```
DELETE FROM stackpatch WHERE foobar != 'FIXED';

ERROR:  column "foobar" does not exist
LINE 1: DELETE FROM stackpatch WHERE foobar != 'FIXED';
                                     ^
```

Reply:
The only solution I can think of is not adding an UNDO block, so these new columns are simply not mapped - would it be better?